examples/js/exporters/GLTFExporter.js

```diff
@@ -707,8 +709,13 @@ THREE.GLTFExporter.prototype = {
 
 				var canvas = cachedCanvas = cachedCanvas || document.createElement( 'canvas' );
 
-				canvas.width = image.width;
-				canvas.height = image.height;
+				if ( image.width > options.maxTextureWidth || image.height > options.maxTextureHeight ) {
+
+					console.warn( 'GLTFExporter: Resized too big image.', image );
+				}
```

Comment:
Let's omit the warning here, but change the default max size to `Infinity`. If the user has to opt in there's no need to warn.